src/main/java/fi/tkgwf/ruuvi/config/Config.java

```diff
@@ -197,24 +196,28 @@ private static void readTagNames() {
     }
 
     public static DBConnection getDBConnection() {
+        if (dbConnection != null) {
```

Comment:
Consider inverting the if-statement and instead create the dbConnection inside the if-block to avoid double return statements (common practice for create-if-does-not-exist-yet cases)

Reply:
That would work, and is probably the most clear in the long run considering the amount of possible connection types seem to be growing